tests/test_enterprise/management/test_email_drip_for_missing_dsc_records.py

```diff
@@ -71,14 +71,16 @@ def create_enrollments(self, num_learners, enrollment_date):
                 enterprise_customer_user=enterprise_customer_user,
                 course_id=course_id,
             )
-            enterprise_course_enrollment.created = enrollment_date
+            enterprise_course_enrollment.created = enrollment_time
             enterprise_course_enrollment.save()
 
     def setUp(self):
         super().setUp()
-        today = timezone.now().date()
```

Comment:
This should be a whole DateTime object instead of just Date as we save this value in `created` field.